articles/techniques/going-further.md

```diff
@@ -46,15 +46,15 @@ A map from integers to Paulis, for instance, looks much the same as a map from f
 function MapIntsToPaulis(fn : Int -> Pauli, values : Int[]) : Pauli[] {
     mutable mappedValues = new Pauli[Length(values)];
     for (idx in 0..Length(values) - 1) {
-        set mappedValues[idx] = fn(values[idx]);
+        set mappedValues w/= idx <- fn(values[idx];
     }
     return mappedValues;
 }
 
 function MapDoublesToStrings(fn : Double -> String, values : Double[]) : String[] {
     mutable mappedValues = new String[Length(values)];
     for (idx in 0..Length(values) - 1) {
-        set mappedValues[idx] = fn(values[idx]);
+        set mappedValues w/= idx <- fn(values[idx];
```

Comment:
```suggestion
        set mappedValues w/= idx <- fn(values[idx]);
```

Reply:
oh god, I think I need to stop reviewing stuff on the side - thanks @cgranade!